pkg/apis/servicecatalog/checksum/unversioned/checksum.go

```diff
@@ -38,6 +39,10 @@ func InstanceSpecChecksum(spec servicecatalog.InstanceSpec) string {
 	if spec.Parameters != nil {
 		specString += fmt.Sprintf("parameters:\n\n%v\n\n", string(spec.Parameters.Raw))
 	}
+	if spec.AlphaSecretParameters != nil {
+		// TODO: calc checksum of secret contents instead?
```

Comment:
We will need to distinguish, eventually, between the spec checksum and the checksum of the parameters that we sent to the broker.  We will not ever be able to set the checksum of the parameters in the registry, so that will need to be calculated in the controller.